keras_cv/models/classification/image_classifier_presets.py

```diff
@@ -39,4 +42,117 @@
         "weights_url": "https://storage.googleapis.com/keras-cv/models/resnet50v2/imagenet-classifier-v0.h5",  # noqa: E501
         "weights_hash": "77fa9f1cd1de0e202309e51d4e598e441d1111dacb6c41a182b6c63f76ff26cd",  # noqa: E501
     },
+    "efficientnetv2-s_imagenet_classifier": {
```

Comment:
It seems like in the above resnet preset, we use all underscores, but here we have a dash.

How about we do all underscores? (Throughout all the presets in this PR)

Reply:
done